### ISSUE TITLE
PhoneNumbers.dll not found when references / compiled for Xamarin Android

### ISSUE THREAD

**Issue by softlion** [twcclegg/libphonenumber-csharp#42]
I have an error when the nuget is added to a Xamarin Android app through a netstandard project (ie: a netstandard 2.0 project reference the libphonenumber nuget, and the android project reference the netstandard 2 project):

    Xamarin.Android.Common.targets(1397,2): error : Exception while loading assemblies: System.IO.FileNotFoundException: Could not load assembly 'PhoneNumbers, Version=8.8.3.0, Culture=neutral, PublicKeyToken='. Perhaps it doesn't exist in the Mono for Android profile?

I suppose the manifest file should not contain a "runtime" entry, but a "reference" entry instead as explained [here](https://docs.microsoft.com/en-us/nuget/schema/nuspec#dependencies).

        "runtime": {
          "PhoneNumbers.dll": {}
        }

Adding libphonenumber as a direct dependency to the Android project does fix the problem.

**Comment by twcclegg** [comment]
I'm not familiar with including references as dll, only adding as reference
```xml
  <ItemGroup>
    <PackageReference Include="libphonenumber-csharp" Version="8.8.3" />
  </ItemGroup>
```
Is there a reason to add them differently for Xamarin?

**Comment by softlion** [comment]
Xamarin says it’s a problem with msbuild which forget to add the dll to Thebes final project.

Does it work when you create a console project A, a netstandard 2 project B, add lib phone number to B, and add B to A ?

Xamarin says that in that case the dll is not copied / included in project A by msbuild and running A will fail.

**Comment by twcclegg** [comment]
I believe this would have been fixed in v8.9.7. Reopen if not the case and still needed.
